core/modules/startup/render.js

```diff
@@ -21,6 +21,7 @@ exports.synchronous = true;
 // Default story and history lists
 var PAGE_TITLE_TITLE = "$:/core/wiki/title";
 var PAGE_STYLESHEET_TITLE = "$:/core/ui/PageStylesheet";
```

Comment:
Did you make sure, that `$:/core/ui/PageStylesheet` is not used hardcoded anywhere else?